examples/typescript-example/src/App.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { FC, useEffect, useRef } from 'react'
+import {
+  useTable,
+  useGroupBy,
+  useFilters,
+  useSortBy,
+  useExpanded,
+  usePagination,
+  Column,
+  CellProps,
+  UseSortByColumnProps,
+  TableInstance,
+  UseFiltersColumnProps,
+  ColumnInstance,
+  useBlockLayout,
+  useResizeColumns,
+  UseResizeColumnsHeaderProps
+} from 'react-table'
+import styled from 'styled-components'
+import makeData from './makeData'
+
+type Data = object
+
+type Props = {
+  columns: Column<Data>[]
```

Comment:
If I want to add a column to contain one or multiple components, does it work? If it does, how do I do it? For example, I want another column for each row to allow users to click on a button to edit the row or to register for membership. Would appreciate your advice.

Reply:
Also, if I wanna add links to the rows, how do you suggest I do that?

Reply:
You'd do far better asking on https://spectrum.chat/react-table rather than adding unrelated questions to some unrelated bug.

Reply:
sorry to have annoyed you. If I had known about that forum, I wouldn't be asking questions here.

Reply:
It's listed in the contributing guide, should probably be on the main page too to be honest.

Reply:
Also, just giving an unwanted advice. You probably should figure out how to block unrelated questions if you don't wanna see more of them.